### PR TITLE
Fix: add list of acceptable documents types to upload

### PR DESCRIPTION
**Pull request by dhanuwat**
Apply commit fixes from TrakX:
8158d5257 Fix: update beneficiaries after delete
f314c1530 Fix: upload only allowed types of files

### CODE REVIEW

File: src/components/Beneficiaries/index.tsx [hunk 1]
```diff
@@ -123,6 +128,10 @@ class BeneficiariesComponent extends React.Component<Props, State> {
         if (nextProps.beneficiariesActivateSuccess && !beneficiariesActivateSuccess) {
             this.handleToggleConfirmationModal();
         }
+
+        if (nextProps.beneficiariesDeleteSuccess && !beneficiariesDeleteSuccess) {
+            this.props.beneficiariesFetch();
+        }
```
Comment (josadcha) [inline src/components/Beneficiaries/index.tsx:134]: currently WalletsScreen includes handler for BeneficiaryDelete ( see image)
image

Comment (josadcha) [reply src/components/Beneficiaries/index.tsx:134]: Besides, beneficiary flow has been updated. Now we fetch beneficiary only for selected wallet

**Review by josadcha — CHANGES_REQUESTED** [review]

File: src/components/UploadFile/index.tsx [hunk 2]
```diff
@@ -70,6 +70,7 @@ const UploadFileComponent: React.FC<Props> = ({
                 {label ? <label>{label}</label> : null}
                 <div className="pg-upload-file__content__form">
                     <input
+                        accept="application/pdf,image/jpeg,image/png"
```
Comment (akhlopiachyi) [inline src/components/UploadFile/index.tsx:73]: Add 'accept' to props (should be optional) and use 'application/pdf,image/jpeg,image/png' as default value

File: src/components/Beneficiaries/index.tsx (hunk 1 above)
Comment (dhanuwat) [reply src/components/Beneficiaries/index.tsx]: Fixed, remove extra logic in Beneficiaries component. thanks.

File: src/components/UploadFile/index.tsx (hunk 2 above)
Comment (dhanuwat) [reply src/components/UploadFile/index.tsx]: Fixed, thanks.

**Review by josadcha — APPROVED** [review]

File: src/components/UploadFile/index.tsx [hunk 3]
```diff
@@ -34,6 +35,7 @@ const UploadFileComponent: React.FC<Props> = ({
     id,
     isMobileDevice,
     uploadedFile,
+    accept,
```
Comment (akhlopiachyi) [inline src/components/UploadFile/index.tsx:38]: accept = 'application/pdf,image/jpeg,image/png'

File: src/components/UploadFile/index.tsx [hunk 4]
```diff
@@ -70,6 +72,7 @@ const UploadFileComponent: React.FC<Props> = ({
                 {label ? <label>{label}</label> : null}
                 <div className="pg-upload-file__content__form">
                     <input
+                        accept={accept || "application/pdf,image/jpeg,image/png"}
```
Comment (akhlopiachyi) [inline src/components/UploadFile/index.tsx:75]: accept={accept}

**Comment by akhlopiachyi** [comment]
@dhanuwat Please, rebase

File: src/components/UploadFile/index.tsx (hunk 3 above)
Comment (dhanuwat) [reply src/components/UploadFile/index.tsx]: Fixed, thanks.

File: src/components/UploadFile/index.tsx (hunk 4 above)
Comment (dhanuwat) [reply src/components/UploadFile/index.tsx]: Fixed, thanks.

**Review by akushniruk — APPROVED** [review]

**Pull request merged**